exchanges/stream/websocket_connection.go

```diff
@@ -326,5 +321,19 @@ func (w *WebsocketConnection) SendMessageReturnResponses(ctx context.Context, si
 		w.Reporter.Latency(w.ExchangeName, outbound, time.Since(start))
 	}
 
+	// Only check context verbosity. If the exchange is verbose, it will log the responses in the ReadMessage() call.
+	if request.IsVerbose(ctx, false) {
+		for i := range resps {
+			log.Debugf(log.WebsocketMgr, "%v %v websocket connection: Received response [%d/%d]: %v", w.ExchangeName, removeQuery(w.URL), i+1, len(resps), string(resps[i]))
```

Comment:
I think there's less of a need to `websocket connection` across all these verbose comments now. You can tell its that because ~[of the way it is](https://www.youtube.com/watch?v=_d8mjam7KG8)~ WebsocketMgr and that its `Received response` or `sending message`.